configure.ac

```diff
@@ -413,6 +412,18 @@ then
     PKG_CHECK_MODULES(X11SM, sm ice)
     LIBS=$LIBS_save
     CPPFLAGS=$CPPFLAGS_save
+
+    with_cxx11=no
+    # Enable C++11 std if gtkmm >= 3.18.0
+    PKG_CHECK_MODULES(GTKMM, gtkmm-3.0 >= 3.18.0,
+      [ AX_CXX_COMPILE_STDCXX_11( [noext], [mandatory])
+        with_cxx11=yes
+      ],
+      [ AC_MSG_ERROR(
+	  [You need gtkmm version equal or newer than 3.18.0]
```

Comment:
Workrave should still support gtkmm < 3.18. So I think this error message should be removed.